eth/config.go

```diff
@@ -59,7 +59,8 @@ var DefaultConfig = Config{
 		GasPrice: big.NewInt(params.GWei),
 		Recommit: 3 * time.Second,
 	},
-	TxPool: core.DefaultTxPoolConfig,
+	TxPool:    core.DefaultTxPoolConfig,
+	RPCGasCap: big.NewInt(50000000),
```

Comment:
Is it necessary to use `big.Int` instead of uint64? Perhaps uint64 is enough to cover gas limit?